game_controller.py

```
@@ -1,7 +1,81 @@
+import random
+
+import user_interaction as user_inter
+from citizens.citizen import Citizen
+from citizens.player import Player
+from citizens.spy import Spy
+from game import Game
+
+
 class GameController:
     def __init__(self) -> None:
+
+        self.citizens_dict = {
+            "Охотник": "Ловушка",
+            "Врач": "Антидот",
+            "Сутенер": "Клофелинщица"
+        }
+
+        self.first_user = 'GvinP'
+        self.second_user = 'Runmaget'
+        self.available_citizens = []
+
+    def start_game(self):
+        self.game = Game()
+
+    def prepare_game(self):
+        self.create_citizens()
+        self.set_order()
+        self.create_player()
+        self.create_spy()
+        self.show_game_state()
+
+    def create_citizens(self):
+        for name, card in self.citizens_dict.items():
+            self.game.citizens.append(Citizen(name=name, citizen_card=card))
+            self.available_citizens.append(
+                Citizen(name=name, citizen_card=card))
+
+    def set_order(self):
+        self.user_names = [self.first_user, self.second_user]
+        random.shuffle(self.user_names)
+
+    def create_player(self):
+        first_player_name = self.first_user[0]
+        second_player_name = self.second_user[1]
+
+        random_citizen = self.available_citizens.pop(
+            random.randint(0,
```

What if available_citizens list is empty?
I think we can raise an exception in this case

---

Will do this in the next commit.